soc/arm/nxp_kinetis/k6x/soc.c

```diff
@@ -120,7 +120,7 @@ static ALWAYS_INLINE void clock_init(void)
  * @return 0
  */
 
-static int fsl_frdm_k64f_init(struct device *arg)
+static int k6x_init(struct device *arg)
```

Comment:
This change isn't related to clock defines. Please move it to commit `soc: arm: nxp: kinetis: k6x: add K66F support`

Reply:
@MaureenHelm Thanks, resolved.